src/package/components/Prompt/index.js

```diff
@@ -0,0 +1,39 @@
+import React, {PropTypes} from 'react'
+import {Link} from 'react-router-dom'
+import {startsWith} from 'lodash'
+import Button from 'package/components/Button'
+import Paragraph from 'package/components/Paragraph'
+
+const Prompt = ({
+  description,
+  actionText,
+  action,
+}) => (
+  <div>
+    <Paragraph>
+      {description}
+    </Paragraph>
+    <div className='mt3'>
+      {startsWith(action, '/') 
```

Comment:
Yes that would be cool. Would need some more testing to deal with the children still rendering though with JSX as mentioned ^ for the `Maybe` comment. If you want to work on that that would be awesome. It's lower priority for me :)